src/transformers/tokenization_utils_fast.py

```diff
@@ -278,6 +278,12 @@ def get_added_vocab(self) -> dict[str, int]:
         """
         return {k.content: v for v, k in sorted(self.added_tokens_decoder.items(), key=lambda item: item[0])}
 
+    def __bool__(self) -> bool:
+        """
+        Returns True, to avoid expensive `assert tokenizer` gotcha's.
```

Comment:
```suggestion
        Returns True, to avoid expensive `assert tokenizer` gotchas.
```
Leaving the decision on the change itself to @arthurzucker and @itazap, I'm just diving in here to catch a wayward apostrophe!